web/pages/css/document.css

```diff
@@ -102,3 +103,37 @@
   font-style: italic;
   text-align: center;
 }
+
+/* Override default modal CSS */
+
+#about_modal .modal-header {
```

Comment:
Kind of minor but, would suggest using a class selector over an ID selector for these styles.

Reply:
I think I chose to do ID in this process was due to specificity issues, but your wrapClass suggestion was exactly what I needed.